### PR TITLE
[plugin.video.manoto] v2.1.1

### PR DESCRIPTION
**Pull request by bbk79**
HTML parsing no longer works, using a hard-coded URL that won't change according to the owners.

### Description
HTML parsing no longer works, using a hard-coded URL that won't change according to the owners.
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.manoto/addon.xml [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
-    <import addon="script.module.beautifulsoup" version="3.2.1"/>
   </requires>
   <extension point="xbmc.python.pluginsource"
             library="default.py">
         <provides>video</provides>
   </extension>
   <extension point="xbmc.addon.metadata">
-    <summary>Manoto TV</summary>
-    <description>Manoto TV Addon for Kodi</description>
+    <summary lang="en_US">Manoto TV</summary>
```
Comment (Rechi) [inline plugin.video.manoto/addon.xml]: `lang="en"`

File: plugin.video.manoto/addon.xml [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
+    <description lang="en_US">Manoto TV Addon for Kodi</description>
```
Comment (Rechi) [inline plugin.video.manoto/addon.xml]: `lang="en"`

File: plugin.video.manoto/default.py [hunk 3]
```diff
@@ -10,21 +9,6 @@
...
-
-    if stream is not None and stream['src'] is not None:
-            xbmc.Player().play(stream['src'], li, False)
-    else:
-        xbmcgui.Dialog().ok(addonname,  __language__(30002), __language__(30010))
-except urllib2.HTTPError, err:
-        xbmcgui.Dialog().ok(addonname,  __language__(30000) % err.code, __language__(30010))
-except:
-        xbmcgui.Dialog().ok(addonname,  __language__(30001), __language__(30010))
+xbmc.Player().play(liveVideoUrl, li, False)
```
Comment (Rechi) [inline plugin.video.manoto/default.py]: Calling `xbmc.Player().play(...)` isn't allowed for a plugin.
You have to create a directory listing, which contains one `ListItem`. For that list item you set the path to the above URL.

**Comment by fsmaximum** [comment]
salam Babak,
chetori moshkele Manoto ro bartaraf konam too Kodi?
Ba tashakor

**Comment by bbk79** [comment]
Thanks for the review @Rechi, hope it looks better now!

**Pull request merged**